### PR TITLE
Copy Github Access Token

### PR DESCRIPTION
**Pull request by ioedeveloper**
fixing #2528

### CODE REVIEW

File: src/app/ui/copy-to-clipboard.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ var css = csjs`
   }
 `
 
-module.exports = function copyToClipboard (getContent, tip = 'Copy value to clipboard', icon = 'fa-copy') {
+module.exports = function copyToClipboard (getContent, tip = 'Copied value to clipboard', icon = 'fa-copy') {
```
Comment (LianaHus) [inline src/app/ui/copy-to-clipboard.js]: this is the title(tooltip) of the button. IMHO this should stah "Copy"

File: test-browser/tests/generalSettings.js [hunk 2]
```diff
@@ -57,7 +57,7 @@ module.exports = {
     browser.waitForElementVisible('*[data-id="verticalIconsKindsettings"]')
     .click('*[data-id="copyToClipboardCopyIcon"]')
     .waitForElementVisible('*[data-shared="tooltipPopup"]:nth-last-of-type(1)')
-    .assert.containsText('*[data-shared="tooltipPopup"]:nth-last-of-type(1)', 'Copy value to clipboard')
+    .assert.containsText('*[data-shared="tooltipPopup"]:nth-last-of-type(1)', 'Copied value to clipboard')
```
Comment (LianaHus) [inline test-browser/tests/generalSettings.js]: also here

File: src/app/ui/copy-to-clipboard.js (hunk 1 above)
Comment (ioedeveloper) [reply src/app/ui/copy-to-clipboard.js]: Okay, then we should probably have a separate message for the success pop up.

File: src/app/ui/copy-to-clipboard.js [hunk 3]
```diff
@@ -12,7 +12,7 @@ var css = csjs`
...
   var copyIcon = yo`<i title="${tip}" class="${css.copyIcon} far ${icon}" data-id="copyToClipboardCopyIcon" aria-hidden="true"></i>`
```
Comment (LianaHus) [inline src/app/ui/copy-to-clipboard.js]: since you are on this PR could you please add p-2 to the class list of copyToClipboardCopyIcon here

**Pull request merged**